dart-impl/base/src/hwinfo.c

```diff
@@ -358,9 +358,9 @@ dart_ret_t dart_hwinfo(
     hw.cpu_id = sched_getcpu();
   }
 #else
-  DART_LOG_ERROR("dart_hwinfo: "
-                 "HWLOC or PAPI required if not running on a Linux platform");
-  return DART_ERR_OTHER;
+//  DART_LOG_ERROR("dart_hwinfo: "
```

Comment:
Can we use something like this here? 

https://gerrithub.io/c/geopm/geopm/+/367743/6/src/geopm_sched.c#102

Reply:
Let me check it